src/components/CompleteProfile/CompleteProfile.tsx

```diff
@@ -0,0 +1,241 @@
+					</div>
+
+					<div>
+						<input id='canJoinLocalEvents' name='canJoinLocalEvents' type='checkbox' />
+						<label htmlFor='canJoinLocalEvents'>I can join TorontoJS's local events</label>
+					</div>
+				</div>
+			</details>
+
+			{/* TODO: Image upload */}
+			<details open>
+				<summary className='text-h6'>
+					<span>Avatar:</span>
+				</summary>
+
+				<div className='details-content-wrapper'>
+					<picture>
+						<img hidden />
+					</picture>
+					<div>
+						<label>Upload Product Image:</label>
+						<input type='file' accept='image/png, image/jpeg' />
+					</div>
+				</div>
+			</details>
+
+			<details open>
+				<summary className='text-h6'>
+					<span>More Information:</span>
+				</summary>
+
+				<div className='details-content-wrapper'>
+					<div>
+						<div>
+							<label htmlFor='linkedInProfile'>LinkedIn profile</label>
+							<input id='linkedInProfile' name='linkedInProfile' type='url' />
+						</div>
+						<div>
+							<label htmlFor='githubProfile'>GitHub profile</label>
+							<input id='githubProfile' name='githubProfile' type='url' />
+						</div>
+
+						<div>
+							<label htmlFor='portfolio'>Site/portfolio</label>
+							<input id='portfolio' name='portfolio' type='url' />
+						</div>
+					</div>
+
+					<div>
+						<label htmlFor='skills'>Your skills</label>
+						<textarea id='skills' name='skills' />
+					</div>
+
+					<div>
+						<p>Other social accounts (optional):</p>
+						<div>
+							<button>
+								<Instagram />
+							</button>
+
+							<button>
+								<Threads />
+							</button>
+
+							<button>
+								<Facebook />
+							</button>
+
+							<button>
+								<LinkedIn />
+							</button>
+
+							<button>
+								<XTwitter />
+							</button>
+
+							<button>
+								<BlueSky />
+							</button>
+
+							<button>
+								<DevTo />
+							</button>
+						</div>
+					</div>
+
+					<div>
+						<div>
+							<label htmlFor='threads'>Threads</label>
+							<input id='threads' name='threads' type='url' />
+						</div>
+						<div>
+							<label htmlFor='facebook'>Facebook</label>
+							<input id='facebook' name='facebook' type='url' />
+						</div>
+						<div>
+							<label htmlFor='instagram'>Instagram</label>
+							<input id='instagram' name='instagram' type='url' />
+						</div>
+						<div>
+							<label htmlFor='mastodon'>Mastodon</label>
+							<input id='mastodon' name='mastodon' type='url' />
+						</div>
+						<div>
+							<label htmlFor='twitter'>Twitter</label>
+							<input id='twitter' name='twitter' type='url' />
+						</div>
+						<div>
+							<label htmlFor='bluesky'>Bluesky</label>
+							<input id='bluesky' name='bluesky' type='url' />
+						</div>
+						<div>
+							<label htmlFor='dev-to'>DEV.to</label>
+							<input id='dev-to' name='dev-to' type='url' />
+						</div>
+					</div>
+				</div>
+			</details>
+		</div>
+
+		<Button isPrimary id='submit-button' type='submit'>
```

Comment:
The button probably needs 100% width:

![image](https://github.com/user-attachments/assets/bb418673-8937-410c-aedb-1e94215c3000)

See the Figma design:

![image](https://github.com/user-attachments/assets/384dc40d-2d45-4dd6-aebb-9cbe01ca8bb7)